src/util/key.rs

```diff
@@ -272,16 +272,13 @@ pub struct SortKey(u8, [u8; 32], [u8; 32]);
 
 impl fmt::Display for PublicKey {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        if self.compressed {
-            for ch in &self.inner.serialize()[..] {
-                write!(f, "{:02x}", ch)?;
-            }
-        } else {
-            for ch in &self.inner.serialize_uncompressed()[..] {
+        // TODO: fast hex encoding
```

Comment:
I have no idea how to implement this TODO, could you possibly do something like
- Use `TODO(Kixunil): fast hex encoding` so a dev wishing to do this can pester you about what it means
- Leave it out and write an issue when this merges
- Put more description in the TODO comment

Thanks man

Reply:
I think it's okay -- I also have no idea *how* to do this, but it's clear enough *what* needs to be done, so having a TODO sitting in the codebase seems fine. It could also be an issue, though I would expect this to go unaddressed for a very long time.

Reply:
Basically the same thing as https://github.com/rust-bitcoin/bitcoin_hashes/issues/171

How: allocate a large enough buffer, write the hex into it and then convert it to `&str`. I did a similar thing in `consensus::serde` but I'd like to move these common helpers into `bitcoin-internals`.

Reply:
Ok, if you have a specific plan then I'm on @tcharding's side, that you should put your name in the TODO :P

Reply:
Isn't git blame enough? :) Anyway, I created #1261